### PR TITLE
[WIP] Adding on to the objects media blocks

### PR DESCRIPTION
**Pull request by laurarbunea**
adding on to the objects media blocks

### CODE REVIEW

File: core/static_src/sass/objects/_objects.media.scss [hunk 1]
```diff
@@ -4,48 +4,102 @@
...
 
-.media__body {
-    @include clearfix;
-}
+        .media__image--min-width {
+            min-width: $reasonable-image-size;
+        }
 
+        .media__image--max-width {
+            max-width: $reasonable-image-size;
```
Comment (thibaudcolas) [inline core/static_src/sass/objects/_objects.media.scss]: I'm not sure I understand what is going on in those `min-width` and `max-width` modifiers, could you explain?

File: core/static_src/sass/objects/_objects.media.scss [hunk 2]
```diff
@@ -4,48 +4,102 @@
...
+
 ```
 */
 
-$spacing-media: 1em;
+$reasonable-image-size: 6.25rem;
 
 .media,
 .media--rev,
 .media--middle-align {
```
Comment (thibaudcolas) [inline core/static_src/sass/objects/_objects.media.scss]: In your examples those two modifier classes are always applied on an element that already has `.media.` Should we remove them from this declaration so we don't have the `clearfix` properties applied twice?

File: core/static_src/sass/objects/_objects.media.scss [hunk 3]
```diff
@@ -4,48 +4,102 @@
 Float an image to the left, with a clearfixed block of content to the right.
 
 ```
-<div class="media">
+<h5>Media object</h5>
+<div class="media" style="background-color:#efefef">
+    <img class="media__image" src="https://placekitten.com/150/150" alt="">
+    <div class="media__body">
+        <p>The textLorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla in urna ut justo suscipit interdum in at purus. Nam in gravida dolor. Aliquam erat volutpat. Vivamus quis euismod nunc, eu venenatis leo.
```
Comment (thibaudcolas) [inline core/static_src/sass/objects/_objects.media.scss]: Lorem: forbidden 🙂

File: core/static_src/sass/objects/_objects.media.scss [hunk 4]
```diff
@@ -58,3 +112,44 @@ $spacing-media: 1em;
...
+            margin-bottom: 0;
+        }
+    }
+}
+
+
+
+// Remove float from any image that breaks on small screens
+//[class*='break']this means any class that contains the word 'break'
+ .media[class*='break'] > .media__image {
```
Comment (thibaudcolas) [inline core/static_src/sass/objects/_objects.media.scss]: Please don't use attribute selectors to target classes like this, it makes it overly complicated to look up style declarations for not much benefit.

As a rule of thumb, if you have to write a comment to explain a piece of CSS/Sass syntax, it's probably too complicated.

**Comment by laurarbunea** [comment]
Sorry this pull request is work in progress. I took it from Kiwibank and wanted to clean it up, so thought I would start a pull request just forgot to write a little note, so yeah this isn't ready to merge yet at all.

**Comment by thibaudcolas** [comment]
Just added a [WIP] in the title to reflect this.

**Pull request closed without merging**